code/modules/jobs/job_types/legion.dm

```diff
@@ -246,4 +250,141 @@ Legionary
 		/obj/item/reagent_containers/pill/patch/healingpowder=2, \
 		/obj/item/ammo_box/m44=2, \
 		/obj/item/flashlight/flare/torch=1)
-	box = null
\ No newline at end of file
+	box = null
+
+/datum/job/f13explorer
+	title = "Explorer"
```

Comment:
can you add this to the job console blacklist

Reply:
Done.